synapse/lib/modelrev.py

```diff
@@ -826,22 +830,25 @@ async def save():
             if nodeedits:
                 await save()
 
-    async def _normFormSubs(self, layers, formname):
+    async def _normFormSubs(self, layers, formname, liftprop=None):
 
         meta = {'time': s_common.now(), 'user': self.core.auth.rootuser.iden}
 
         subprops = {}
 
         form = self.core.model.form(formname)
 
+        if liftprop = None:
```

Comment:
This must be a typo. Should probably is `if liftprop is None:`?

Reply:
Yep, good catch thanks!